.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,10 @@
+contact_links:
+  - name: Grails Core Discussions
+    url: https://github.com/grails/grails-core/discussions
```

Comment:
Grails Core Discussions record should be removed; we are not using grails-core/discussions

Reply:
got it --addressed